### ISSUE TITLE
Django 1.10 incompatibility

### ISSUE THREAD

**Issue by jvamvas** [arteria/django-background-tasks#54]
With django 1.9:
```
$ python manage.py process_tasks
.../django/core/management/__init__.py:345: RemovedInDjango110Warning: 
OptionParser usage for Django management commands is deprecated, use ArgumentParser instead
  self.fetch_command(subcommand).run_from_argv(self.argv)
```

Cannot be fixed until <1.8 support is dropped

**Comment by martinogden** [comment]
Updated management command argument parsing for Django 1.10  [here](https://github.com/arteria/django-background-tasks/pull/56).

**Comment by 029xue** [comment]
Just met the same error as below:
-----
  File "C:\Dev\envs\django\lib\site-packages\background_task\management\commands\process_tasks.py", line 9, in <module>
    class Command(BaseCommand):
  File "C:\Dev\envs\django\lib\site-packages\background_task\management\commands\process_tasks.py", line 13, in Command
    option_list = BaseCommand.option_list + (
AttributeError: type object 'BaseCommand' has no attribute 'option_list'

**Comment by 029xue** [comment]
> BaseCommand.option_list¶
This is the list of optparse options which will be fed into the command’s OptionParser for parsing arguments.

Deprecated since version 1.8:
You should now override the add_arguments() method to add custom arguments accepted by your command. See the example above.

> 

According to this, the code need update to support 1.10.

**Comment by jvamvas** [comment]
Fixed with v1.0.13
